modules/brew/brew.sh

```diff
@@ -212,6 +217,12 @@ DefaultLimitNOFILE=4096:524288
 EOF
 fi
 
+# Disable homebrew analytics if the flag is set to false
+# like secureblue: https://github.com/secureblue/secureblue/blob/live/config/scripts/homebrewanalyticsoptout.sh
+if [[ "${BREW_ANALYTICS}" == false ]]; then
+    echo "HOMEBREW_NO_ANALYTICS=1" >> /usr/etc/environment
```

Comment:
I think there could be a better way to set environment variable to be only available when in terminal session in `profiles.d`. Similarly to how is brew path preferred only when in terminal session.

Making it available globally in `/usr/etc/environment` is not the ideal solution imo.

Reply:
Hmm, a better solution would be to maybe alias `brew` to always launch with `HOMEBREW_NO_ANALYTICS` set up.

Having it applied in terminal session only is flawed, since I forgot that brew runs in background to update & upgrade.

Reply:
If we could fix this error:

`#46 14.06 su: user linuxbrew does not exist or the user entry does not contain all the required fields`

I think that we could just execute

`  su -c "/home/linuxbrew/.linuxbrew/bin/brew analytics off" -s /bin/bash linuxbrew`

to turn off analytics. This would be the best way, since it does not touch environment & shell, but the app itself.

Reply:
I ended up just improving the script a bit, to account for existing homebrew_analytics variable, so it doesn't get duplicated.

I think that there is not a better solution for doing this, so I'll mark this as resolved.